testez-cli/src/tools/roblox_cli.rs

```diff
@@ -19,10 +16,15 @@ pub fn roblox_cli_run(place_path: &Path, entrypoint_path: &str, as_core_script:
         &place_arg,
         "--entrypoint",
         entrypoint_path,
-        "--assetFolder",
-        &content_path_arg,
     ]);
 
+    let install = RobloxStudio::locate();
+    if let Ok(install_location) = install {
```

Comment:
```suggestion
    if let Ok(install_location) = RobloxStudio::locate() {
```

Can we inline this condition? I don't think we use `install` anywhere else.